token/services/db/driver/token.go

```diff
@@ -128,12 +130,12 @@ type TokenDB interface {
 	IsMine(txID string, index uint64) (bool, error)
 	// UnspentTokensIterator returns an iterator over all owned tokens
```

Comment:
Maybe it's best to also make clear that the UnspentTokensIterator also includes partially owned tokens. If we don't want to change the API, it would suffice to do it in the comment (and also in the UnspentTokensIteratorBy). Maybe something like this?

```
	// UnspentTokensIterator returns an iterator over all tokens that are fully or partially owned by any wallet.
```

Reply:
okay